docs/readmes/bazel/agw_with_bazel.md

```diff
@@ -136,13 +138,27 @@ The Docker Bazel caches are Docker containers that contain pre-built Bazel cache
 
 The Docker Bazel caches were introduced in [#14562](https://github.com/magma/magma/issues/14562) - this issue also contains a detailed slide with information on the Docker Bazel cache setup.
 
+#### GitHub cache
+
+GitHub Actions provides a generic [built-in caching solution](https://github.com/actions/cache), which can be used to store the Bazel disk cache. Possible downsides of this caching approach, as with the [Docker caches](#docker-bazel-cache), are that the cache is not managed by Bazel and has to be kept up-to-date and below the maximum cache size.
```

Comment:
"the maximum cache size" is not mentioned before.
Suggestion:
... and has to be kept up-to-date and below a reasonable cache size. The GitHub caches are restricted to 10GB and are currently used by Magma for additionally purposes, e.g., caching Vagrant base images.

Reply:
Updated the section.